compiler/optimizer/RegisterCandidate.hpp

```diff
@@ -50,19 +47,20 @@ namespace TR
 class GlobalSet
    {
 public:
-   GlobalSet(TR::Compilation * comp, const TR::Allocator & alloc);
+   GlobalSet(TR::Compilation * comp, TR::Region &region);
 
    class Set;
    Set * operator[](uint32_t blockNum)
    {
     if(blockNum == TR::CFG::StartBlock || blockNum == TR::CFG::EndBlock)
         return NULL;
-    return _refAutosPerBlock.ElementAt(blockNum);
+    
+    return _refAutosPerBlock[blockNum];
    }
 
    void collectReferencedAutoSymRefs(TR::Block * BB);
-   bool isEmpty() { return _refAutosPerBlock.IsEmpty(); }
-   void makeEmpty() { _refAutosPerBlock.MakeEmpty(); }
+   bool isEmpty() { return _refAutosPerBlock.empty(); }
+   void makeEmpty() { _refAutosPerBlock.clear(); }
    void initialize(uint32_t numSymRefs, int32_t numBlocks) {_maxEntries = numSymRefs * numBlocks;}
 
    //Set, SparseSet and DenseSet
```

Comment:
I'm open to seeing if there is a better solution. For now I think this will do, but we can consider it. I believe these are all using bitvectors so that might be an advantage in this particular case.